Lib/imaplib.py

```diff
@@ -904,7 +904,11 @@ def unsubscribe(self, mailbox):
 
 
     def unselect(self):
-        """Close the mailbox without expunging it.
+        """The UNSELECT command frees server's resources associated with the
```

Comment:
I don't see other docstrings repeating the command name. Maybe:

```suggestion
        """Free server's resources associated with the
```